<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2e2e2e;        /* Dark gray */
border: 3px solid #dcdcdc;        /* Light gray outline */
border-radius: 16px;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblAngleMode">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background: transparent;
color: gray;
font-weight: bold;
</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>DEG</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>331</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="btnSin">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #81c784;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>sin</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnCos">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #81c784;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>cos</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnTan">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #81c784;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>tan</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btnSqrt">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #81c784;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>√</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="btnPow">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #81c784;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>xⁿ</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btnOpen">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #81c784;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnClose">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #81c784;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnFrac">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #81c784;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>Frac</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btnDeg">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #81c784;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>Deg</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="btnRad">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #81c784;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>Rad</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btn7">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #666666;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btn8">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #666666;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btn9">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #666666;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btnDel">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #e57373;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>DEL</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="btnAC">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #e57373;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>CA</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btn4">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #666666;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btn5">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #666666;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btn6">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #666666;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="btnMul">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #f39c12;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="btnDiv">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #f39c12;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>÷</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="btn1">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #666666;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btn2">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #666666;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="btn3">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #666666;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="btnAdd">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #f39c12;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="btnSub">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #f39c12;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="btn0">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #666666;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="btnDot">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #666666;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="btnEng">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #9b59b6;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>ENG</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="btnAns">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #9b59b6;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>ANS</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="btnEqual">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #64b5f6;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #cccccc;
    border-radius: 10px;
    padding: 12px;
}

QPushButton:hover {
    background-color: #505050;
    border-color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="display">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2c2c2c;
color: #ffffff;
font-size: 24px;
font-weight: bold;
font-family: 'Courier New', monospace;
border: 2px solid #aaaaaa;
border-radius: 10px;
padding: 8px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>display</zorder>
   <zorder>lblAngleMode</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
